src/Appwrite/Specification/Format/Swagger2.php

```diff
@@ -225,6 +225,10 @@ public function parse(): array
                     'required' => !$param['optional'],
                 ];
 
+                if($name == 'queries') {
```

Comment:
We should never trust a variable/parameter name for business logic. What we can do is leverage the param instance type for differentiation, just like we do below this condition. 👇

Reply:
Instance type? the param instance type I think is the validator type right? Which in this case is ArrayList, how will that help me determine if it's a query?